master/static/srv/get_experiment.sql

```diff
@@ -13,7 +24,18 @@ SELECT
     e.notes AS notes,
     e.start_time AS start_time,
     e.end_time AS end_time,
-    'STATE_' || e.state AS state,
+    CASE
+      WHEN e.state = 'PAUSED'
+        THEN 'STATE_PAUSED'
+      WHEN e.state = 'CANCELED'
+        THEN 'STATE_CANCELED'
+      WHEN (e.config->'resources'->>'slots_per_trial')::int > max_slots.count
+        THEN 'STATE_INSUFFICIENT'
```

Comment:
Can we please discuss it in backend office hours or a dedicated meeting? I am not sure this is good long-term plan to implement this feature.

Reply:
I'd like to listen in if/when this happens ✋🏼

Reply:
ok, I will plan to move Insufficient to be more like Queued which has to come from Resource Manager.
I created a new PR today #4524 to prevent these experiments from being created in the first place